enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -692,13 +697,18 @@ func (c *ociContainer) createSpec(ctx context.Context, cmd *repb.Command) (*spec
 		return nil, fmt.Errorf("get container user: %w", err)
 	}
 
-	cpuSpecs := &specs.LinuxCPU{}
+	var milliCPUHardLimit int64
+	if *cpuRelativeLimit >= 0 {
+		milliCPUHardLimit = c.milliCPU + int64(*cpuRelativeLimit)*1000
+	}
 	if *cpuLimit != 0 {
```

Comment:
This wasn't handled before, but what happens if this flag is negative? Maybe that should just be ignored.

Reply:
Changed to `> 0`